Composer/packages/server/src/controllers/asset.ts

```diff
@@ -16,6 +18,57 @@ async function getProjTemplates(req: Request, res: Response) {
   }
 }
 
+export async function getProjTemplatesV2(req: any, res: any) {
+  try {
+    let templates: BotTemplateV2[] = [];
+
+    // Get FeedUrl
+    const { feedUrls, getFirstPartyNpm } = req.body;
+
+    // Grab templates from FeedURls
+    if (feedUrls) {
+      // TODO: Current assumption is that the feed is an npm feed with given result structure (https://registry.npmjs.org/-/v1/search?text=docker&size=100&from=0&quality=0.65&popularity=0.98&maintenance=0.5)
```

Comment:
what's the future plan here with this TODO?

Reply:
Ive removed the TODO, the work needed for that revolved around documentation tasks that have been created and are being tracked. Essentially we just need to let folks know through the docs the standard expected structure of the template feeds once we allow custom feeds for templates (post R12). 
The expected structure is set by the respective package manager (npm) so we are not creating our own feed schema or anything, we are sticking to the format that  npm dictates to not uneccisarily reinvent the wheel. 

I have removed the TODO.